common/networkProfile.go

```diff
@@ -0,0 +1,30 @@
+package common
+
+// NetworkProfile is used in ThrottleNetwork.
+type NetworkProfile struct {
+	// Minimum latency from request sent to response headers received (ms).
+	Latency float64 `js:"latency"`
+
+	// Maximal aggregated download throughput (bytes/sec). -1 disables download throttling.
+	DownloadThroughput float64 `js:"downloadThroughput"`
```

Comment:
This is a bit verbose. Should we just say `Download` as the comment makes it clear? And it's in a `NetwrokProfile` already-also clear from the context. No strong opinion.

Reply:
Thanks, resolved. See [comment](https://github.com/grafana/xk6-browser/pull/1094#discussion_r1388140886) for details.